deploy/backplane/srep/10-srep-admins-project.ClusterRole.yml

```diff
@@ -102,3 +102,21 @@ rules:
   - serverstatusrequests
   verbs:
   - '*'
+# SRE can delete csv, installplans, and subscriptions
+- apiGroups:
+  - operators.coreos.com
+  resources:
+  - clusterserviceversions
+  - installplans
+  - subscriptions
+  verbs:
+  - delete
+# SRE can create/update/patch subscriptions
```

Comment:
I'd like for us to not need to recreate subscriptions.

I'm asking a question of the hive team to see if a forced cluster sync is a better way of doing this.

/hold
pending hive team's answer

Reply:
From hive team, deleting the `clustersync` can force a re-sync. In general, we can wait 2 hours for hive to auto re-sync the subscription, as subscription is not an urgent thing usually.

I will update the RBAC to remove the permission for recreating `subscription`, and add permission for SRE to perform force resync in backplane (if not yet).

Please let me know if there's any concern. Thanks!